docs/UserGuide.md

```diff
@@ -308,19 +323,19 @@ Format: `find_add KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find_add geylang` returns all users whose addresses contain `geylang`.
 
-![edit format](images/findadd0.png)
+![edit format](images/findadd-UG/findadd0.png)
 
 * Precise expected outputs on success:
 * Successful find message. ‘X persons listed!’’, where X is the number of contacts listed.
```

Comment:
Looks like there's a small typo here with two apostrophes for X persons listed! ' '